styles/globals.css

```diff
@@ -26,7 +26,8 @@ h6 {
 input[type="checkbox"]:focus-visible,
 [role="checkbox"]:focus-visible,
 [role="radio"]:focus-visible,
-[role="combobox"]:focus-visible {
+[role="combobox"]:focus-visible,
+[role="switch"]:focus-visible {
```

Comment:
This allows for switches to keep the uniform focus styling we've been working on.